library/core/src/ffi/c_str.rs

```diff
@@ -165,13 +165,15 @@ impl fmt::Display for FromBytesUntilNulError {
 
 #[stable(feature = "cstr_debug", since = "1.3.0")]
 impl fmt::Debug for CStr {
+    #[inline]
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
         write!(f, "\"{}\"", self.to_bytes().escape_ascii())
```

Comment:
I'm torn on this one.  Notably, for structs we even send them through `dyn Debug` for their fields, so it feels odd to inlinehint the formatting stuff here.

How strongly do you feel about this one?  All the other ones I agree with completely.

Reply:
Sounds reasonable to remove it then, I'll update